{
  "date": "26-01-1936",
  "source": "Вечерняя Москва",
  "topic": "Ошибка обработки колонки",
  "summary": "В тексте представлена информация об ошибке, возникшей при обработке колонки.",
  "content": "[Ошибка при обработке колонки: <AioRpcError of RPC that terminated with:\n\tcode = StatusCode.INVALID_ARGUMENT\n\tdetails = \"Error in session internal_id=146c08d-51205a4a-eb73994d-e61d5827&request_id=942cc488-5a8c-4d0b-b26b-893e0472b281&client_request_id=3ae20ee6-266f-4c4f-b247-7d92fc6a8b29&folder_id=b1giflegi2ke62rs80la: empty message text\"\n\tdebug_error_string = \"UNKNOWN:Error received from peer ipv4:158.160.54.160:443 {created_time:\"2025-04-24T03:37:44.542907895+00:00\", grpc_status:3, grpc_message:\"Error in session internal_id=146c08d-51205a4a-eb73994d-e61d5827&request_id=942cc488-5a8c-4d0b-b26b-893e0472b281&client_request_id=3ae20ee6-266f-4c4f-b247-7d92fc6a8b29&folder_id=b1giflegi2ke62rs80la: empty message text\"}\"\n\tendpoint = \"llm.api.cloud.yandex.net:443\"\n\tstub_class = TextGenerationServiceStub\n\tx-client-request-id = \"3ae20ee6-266f-4c4f-b247-7d92fc6a8b29\"\n>]",
  "entities": [
    "AioRpcError",
    "StatusCode.INVALID_ARGUMENT",
    "TextGenerationServiceStub",
    "llm.api.cloud.yandex.net"
  ]
}